packages/react/src/Spotlight/Spotlight.tsx

```diff
@@ -18,8 +19,12 @@ export type SpotlightProps = {
 } & PropsWithChildren<HTMLAttributes<HTMLElement>>
 
 export const Spotlight = forwardRef(
-  ({ children, className, as: Tag = 'div', color = 'blue', ...restProps }: SpotlightProps, ref: any) => (
-    <Tag {...restProps} ref={ref} className={clsx('ams-spotlight', `ams-spotlight--${color}`, className)}>
+  ({ as: Tag = 'div', children, className, color = defaultColor, ...restProps }: SpotlightProps, ref: any) => (
+    <Tag
+      {...restProps}
+      ref={ref}
+      className={clsx('ams-spotlight', { [`ams-spotlight--${color}`]: color !== defaultColor }, className)}
```

Comment:
This condition is a little hard to read imo. How about:

```suggestion
      className={clsx('ams-spotlight', { color !== defaultColor && `ams-spotlight--${color}` }, className)}
```